src/cc/frontends/clang/kbuild_helper.cc

```diff
@@ -107,4 +113,81 @@ int KBuildHelper::get_flags(const char *uname_machine, vector<string> *cflags) {
   return 0;
 }
 
+static inline int file_exists(const char *f)
+{
+  struct stat buffer;
+  return (stat(f, &buffer) == 0);
+}
+
+static inline int proc_kheaders_exists(void)
+{
+  return file_exists(PROC_KHEADERS_PATH);
+}
+
+static inline int extract_kheaders(std::string &dirpath, struct utsname &uname_data)
```

Comment:
nit: const std::string& ? same for uname_data

Reply:
ok